addons/website/tests/test_views.py

```diff
@@ -811,6 +811,27 @@ def test_module_new_inherit_view_on_parent_already_forked(self):
         all_title_updated = specific_view.website_meta_title == self.base_view.website_meta_title == "A bug got fixed by updating this field"
         self.assertEqual(all_title_updated, True, "Update on top level generic views should also be applied on specific views")
 
+    def test_module_new_inherit_view_on_parent_already_forked_INSTALL(self):
+        """ This test is following the previous one, it basically test the flow
+            during module install which is altered by condition on `self.pool._init`.
+            See commit message.
+        """
+        # Indirectly simulate module install, this will be enough to test code behavior
+        Website = self.env['website']
+        View = self.env['ir.ui.view']
+        View.pool._init = True
```

Comment:
I don't like that but it will do the job.
Could probably extract the test in a method that is ran during install (not post_install) but that would be annoying to debug